### ISSUE TITLE
Missing bannediplist file under example.group on e2guardian 5.4.3 / Freebsd 12 

### ISSUE THREAD

**Issue by marcelloc** [e2guardian/e2guardian#695]
Hi, I'm updating freebsd port for e2guardian 5.4.3

I'm getting this error:

```
Making install in example.group
Making install in .
/bin/sh /usr/ports/www/e2guardian5/work/e2guardian-5.4.3r/install-sh -d /usr/ports/www/e2guardian5/work/stage/usr/local/etc/e2guardian/lists/example.group &&  for l in addheaderregexplist  bannedsearchoveridelist  bannedextensionlist  bannediplist  bannedmimetypelist  bannedphraselist  oldbannedphraselist  bannedregexpheaderlist  bannedregexpurllist  bannedregexpuseragentlist  bannedsearchlist  bannedsiteiplist  bannedtimelist  exceptionvirussiteiplist  bannedsitelist  bannedsslsiteiplist  bannedsslsitelist  bannedurllist  blankettimelist  contentregexplist  exceptionextensionlist  exceptionvirusextensionlist  exceptionfilesiteiplist  exceptionfilesitelist  exceptionfileurllist  exceptionclientlist  exceptionmimetypelist  exceptionphraselist  oldexceptionphraselist  exceptionregexpurllist  exceptionregexpuseragentlist  exceptionsiteiplist  exceptionsitelist  exceptionregexpheaderlist  exceptionurllist  greysiteiplist  greysitelist  greysslsiteiplist  greysslsitelist  greyurllist  headerregexplist  localbannedsearchlist  localbannedsiteiplist  localbannedsitelist  localbannedsslsiteiplist  localbannedsslsitelist  localbannedurllist  localexceptionsiteiplist  localexceptionsitelist  localexceptionurllist  localgreysiteiplist  localgreysitelist  localgreysslsiteiplist  localgreysslsitelist  localgreyurllist  logregexpurllist  logsiteiplist  logsitelist  logurllist  nocheckcertsiteiplist  nocheckcertsitelist  refererexceptionsiteiplist  refererexceptionsitelist  refererexceptionurllist  responseheaderregexplist  sslsiteregexplist  urlredirectregexplist  urlregexplist  ipnobypass  domainsnobypass  urlnobypass  weightedphraselist  oldweightedphraselist  README ; do  echo "install  -m 0644 $l /usr/ports/www/e2guardian5/work/stage/usr/local/etc/e2guardian/lists/example.group/$l";  install  -m 0644 $l /usr/ports/www/e2guardian5/work/stage/usr/local/etc/e2guardian/lists/example.group/$l;  done
install  -m 0644 addheaderregexplist /usr/ports/www/e2guardian5/work/stage/usr/local/etc/e2guardian/lists/example.group/addheaderregexplist
install  -m 0644 bannedsearchoveridelist /usr/ports/www/e2guardian5/work/stage/usr/local/etc/e2guardian/lists/example.group/bannedsearchoveridelist
install  -m 0644 bannedextensionlist /usr/ports/www/e2guardian5/work/stage/usr/local/etc/e2guardian/lists/example.group/bannedextensionlist
install  -m 0644 bannediplist /usr/ports/www/e2guardian5/work/stage/usr/local/etc/e2guardian/lists/example.group/bannediplist
install: bannediplist: No such file or directory
*** Error code 71

```

Is bannediplist used on example.group or the **install-sh -d /usr/ports/www/e2guardian5/work/stage/usr/local/etc/e2guardian/lists/example.group** reffer to a non used filed?

I've tried with v5.4.3r and v.5.4 branch/tag

**Comment by philipianpearce** [comment]
HI really sorry for the delay in picking this up - fixed in latest v5.4 branch

Regards
Philip

**Comment by philipianpearce** [comment]
Wrong file fixed - should have been Makefile.am
